<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>220</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>220</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>简单绘图</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>301</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="3" column="0">
      <widget class="QPushButton" name="pushButton_circle">
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>27</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>黑体</family>
        </font>
       </property>
       <property name="text">
        <string>中点圆</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="pushButton_DDA">
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>27</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>黑体</family>
        </font>
       </property>
       <property name="text">
        <string>DDA</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="pushButton_Bresenham">
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>27</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>黑体</family>
        </font>
       </property>
       <property name="text">
        <string>Bresenham</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="pushButton_ellipse">
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>27</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>黑体</family>
        </font>
       </property>
       <property name="text">
        <string>中点椭圆</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>300</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
